EmailNotifications/Email.cs

```diff
@@ -39,36 +39,45 @@ private static void SendCompletedCallback(object sender, AsyncCompletedEventArgs
             mailSent = true;
         }
 
-       
-
+  
         public void sendEmail()
         {
-            string[] styleProperties = { "color:red;padding-top: 40px", "color:red;font-weight:bold", "color:black;padding-top: 11px",
-                                            "color:red;padding-top: 35px","color:black","color:black;padding-top: 20px",
+            string[] styleProperties = { "color:red;padding-top: 40px;font-weight: bold", "color:red;font-weight:bold", "color:black;padding-top: 11px",
+                                            "color:red;padding-top: 35px;font-weight: bold","color:black","color:black;padding-top: 20px",
                                                "color:grey" ,"color:grey","color:red;padding-top: 24px","color:red;","color:grey;padding-top: 9px"};
             string[] fontSizeProperties = {"6" ,"3","3","6","4","6","4","4","2","1","3" };
-
             System.Text.StringBuilder sb = new System.Text.StringBuilder();
             using (TableStructure.Table table = new TableStructure.Table(sb, id: "some-id",align:"center"))
             {
           
                 table.StartBody();
                 int count = 0;
-                
-                string[] items = { values[0] + "is " + values[1], "("+ values[2]+")","This problem has been unresolved for " + values[3]+" days",
-                                    "R " + values[4], "Has been lost","R"+ values[5]," is being lost!", "per hour","Resolve issue or precess","Here","call third party help: 011111929292"};
-                foreach (var alert in items)
+                // Line for debugging, will not remove
+                string[] items = { values[0] + " has a " + values[1], "("+ values[2]+")","This problem has been unresolved for " + values[3]+" days",
+                                    "R " + values[4], "Has been lost",values[6]+"L or R "+ values[5]," is being lost!", "per hour","Resolve issue or precess","www.google.co.za","call third party help: 011111929292"};
+                foreach (var alert in values)
                 {
                     using (var tr = table.AddRow(classAttributes: "someattributes"))
                     {
-                        tr.AddCell(alert,align:"center",style: styleProperties[count], fontSize:fontSizeProperties[count]);
+                        if (count== 9)
```

Comment:
What does 9 mean in this context rather create a variable that equal to 9 that gives more meaning and understanding to why 9 is so important